src/browser/api_protocol/api_handlers/window.js

```diff
@@ -41,7 +41,7 @@ module.exports.windowApiMap = {
     'get-window-info': getWindowInfo,
     'get-window-native-id': { apiFunc: getWindowNativeId, apiPath: '.getNativeId' },
     'get-window-options': getWindowOptions,
-    'get-window-preload-script': getWindowPreloadScript,
+    'get-window-preload-scripts': getWindowPreloadScripts,
```

Comment:
yes it does but I was assuming this was for internal use only and that adapters should never call this themselves — not documented so how would they even know